### DISCUSSION TITLE
How to skip minification of one file?

### DISCUSSION THREAD

**Discussion by dangayle** [webpack/webpack#15553] [General]
I know it is easy to turn minification off globally, but is there a way to annotate a file to prevent it from getting minified by the bundler?

**Comment by alexander-akait** [comment]
If you want to ignore minication of chunk you can use the `test` option (look at terser-webpack-plugin docs), if you want to avoid minication module in chunk it is impossible, you can ask about it here https://github.com/terser/terser, terser minify js, so control what should be keep or minimized decides terser
